addons/arsenal/functions/fnc_refresh.sqf

```diff
@@ -25,6 +28,21 @@ if (_updateItems) then {
     call FUNC(updateUniqueItemsList);
```

Comment:
As we are going to be updating the display if _updateVirtualItems is true, no need to do something expensive twice.
```suggestion
    if (!_updateVirtualItems) then {
        call FUNC(updateUniqueItemsList);
    };
```